serverless.yml

```diff
@@ -206,8 +213,8 @@ custom:
     external: ['knex', 'pg', 'sharp']
     bundle: true
     minify: ${self:custom.minifyBuild.${self:provider.stage}}
-    watch:
-      ignore: ['.esbuild', 'dist', 'node_modules', '.build', 'static', '*.test.js']
+    # watch:
+    #   ignore: ['.esbuild', 'dist', 'node_modules', '.build', 'static', '*.test.js']
```

Comment:
Was this causing issues?

Reply:
I was testing something and forgot I had it commented out. Reverted the change